cypress/support/pages/course/pageCourses.js

```diff
@@ -180,18 +212,40 @@ class Courses {
       ])
       .then(interceptions => {
         expect(interceptions[0].response.statusCode).to.equal(200)
-        expect(interceptions[1].state).to.equal('Complete')
         expect(interceptions[1].response.statusCode).to.equal(200)
+        expect(interceptions[2].state).to.equal('Complete')
+        expect(interceptions[2].response.statusCode).to.equal(200)
       })
   }
 
-  taskIsNotVisibleOnCoursePage (taskTitle) {
-    // cy.reload() // Reload is necessary because after deletion of a content element a message window with its title stays hidden in the DOM
-    // cy.url().should('include', '/rooms/')
-    // cy.wait(1000)
-    cy.contains(taskTitle).should('not.exist')
-    // cy.get(Courses.#mainContent).should('not.contain', taskTitle)
-    // cy.get(Courses.#mainContent).contains(taskTitle).should('not.exist')
+  contentIsNotVisibleOnCoursePage (contentTitle) {
+    cy.reload() // Reload is necessary because after deletion of a content element a message window with its title stays hidden in the DOM
+      .wait([
+        '@board_api',
+        '@runtime_config_api',
+        '@public_api'
+      ])
+      .then(interceptions => {
+        expect(interceptions[0].response.statusCode).to.equal(200)
+        expect(interceptions[1].response.statusCode).to.equal(200)
+        expect(interceptions[1].response.statusCode).to.equal(200)
+      })
+    cy.contains(contentTitle).should('not.exist')
+  }
+
+  contentIsNotVisibleOnCoursePage (contentTitle) {
```

Comment:
I guess here you duplicate the method `contentIsNotVisibleOnCoursePage`

Reply:
yes, I removed one.